python/pyspark/sql/tests/connect/test_connect_basic.py

```diff
@@ -21,6 +21,7 @@
 
 import grpc  # type: ignore
 
+from pyspark.sql.connect.column import Column
```

Comment:
This line should be under `if have_pandas`. Here is a follow-up PR.
https://github.com/apache/spark/pull/38976